analyzer/testdata/src/checkers-default/contains/contains_test.go.golden

```diff
@@ -0,0 +1,30 @@
+// Code generated by testifylint/internal/testgen. DO NOT EDIT.
+
+package contains
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestContainsChecker(t *testing.T) {
+	var s = "abc123"
+
+	// Invalid.
+	{
+		assert.NotContains(t, s, "456")                                                // want "contains: use assert\\.NotContains"
+		assert.NotContainsf(t, s, "456", "msg with args %d %s", 42, "42")              // want "contains: use assert\\.NotContainsf"
+		assert.Contains(t, s, "abc123")                                                // want "contains: use assert\\.Contains"
+		assert.Containsf(t, s, "abc123", "msg with args %d %s", 42, "42")              // want "contains: use assert\\.Containsf"
+	}
+
+	// Valid.
+	{
+		assert.Contains(t, s, "abc123")
```

Comment:
You could also mention the assert.Contains (t, err.Error(), "whatever") in ignored as it's currently plan to be in #47

Reply:
Also

https://github.com/search?q=%22assert.%22+%22strings.Contains%22+path%3A_test.go&type=code

You have to add to ignored section, because it should be handled by bool-compare (even if this one is before compare in linter order)

https://github.com/upyun/upx/blob/b184a7cb7c10247456cb9134889c91a3c879e12d/fscmds_test.go#L60